source/uwp/Renderer/lib/AdaptiveTextInputRenderer.cpp

```diff
@@ -41,8 +41,14 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
             bool isMultiline = adaptiveTextInput.IsMultiline();
             winrt::TextInputStyle style = adaptiveTextInput.TextInputStyle();
             isMultiline &= style != winrt::TextInputStyle::Password;
-            textBoxParentContainer = render_xaml::ActionHelpers::HandleInlineAction(
+            auto textBoxWithInlineAction = render_xaml::ActionHelpers::HandleInlineAction(
```

Comment:
is a warning raised to the user that the inline action will not activate when clicked?

Reply:
Yes, we add a warning in `HandleInlineAction`

https://github.com/microsoft/AdaptiveCards/blob/38b8f0934eeb5900f3d948a9691a94376af576b3/source/uwp/SharedRenderer/lib/ActionHelpers.cpp#L355-L358